include/dxc/Support/dxcapi.use.h

```diff
@@ -74,33 +112,24 @@ class DxcDllSupport {
   }
 
 public:
-  DxcDllSupport() : m_dll(nullptr), m_createFn(nullptr), m_createFn2(nullptr) {}
-
-  DxcDllSupport(DxcDllSupport &&other) {
-    m_dll = other.m_dll;
-    other.m_dll = nullptr;
-    m_createFn = other.m_createFn;
-    other.m_createFn = nullptr;
-    m_createFn2 = other.m_createFn2;
-    other.m_createFn2 = nullptr;
-  }
+  SpecificDllLoader()
+      : m_dll(nullptr), m_createFn(nullptr), m_createFn2(nullptr) {}
 
-  ~DxcDllSupport() { Cleanup(); }
+  ~SpecificDllLoader() override { Cleanup(); }
 
   HRESULT Initialize() {
     return InitializeInternal(kDxCompilerLib, "DxcCreateInstance");
   }
 
-  HRESULT InitializeForDll(LPCSTR dll, LPCSTR entryPoint) {
+  HRESULT OverrideDll(LPCSTR dll, LPCSTR entryPoint) {
```

Comment:
This came from a suggestion of mine, but now that the API has been simplified a bit and this only in SpecificDllLoader I find many of the updated users of this to be confusing. We often create the SpecificDllLoader, don't call Initialize() at all, and then call `OverrideDll` - this isn't really overriding, it's just initializing. I think I steered you wrong and the original `InitializeForDll` name was better.